web/beacon-app/src/components/layout/Topbar/Topbar.tsx

```diff
@@ -24,10 +29,24 @@ function Topbar({ Breadcrumbs: CustomBreadcrumbs, isOnboarded, profileData }: To
     window.location.href = '/';
   };
 
+  const [isOpen, setIsOpen] = useState(false);
+
+  const onOpenChange = () => {
+    setIsOpen(!isOpen);
+  };
+
+  const appState = useOrgStore((state: any) => state) as any;
+  const { organizations } = useFetchOrganizations();
+
+  const { menuItems: dropdownItems } = useDropdownMenu({
+    organizationsList: organizations?.organizations,
+    currentOrg: appState?.orgID,
+  });
+
   return (
     <>
       <Header className="flex flex-col-reverse items-baseline justify-center gap-2 bg-[#1D65A6] py-2 md:ml-[250px] md:min-h-[60px] md:border-b md:bg-white">
-        <div className="flex w-11/12 justify-between">
+        <div className="flex w-[98%] justify-between xl:w-[92.5%]">
```

Comment:
A follow on story will need to be created to handle the width because as the screen gets wider the icons move further to the right.